spring-boot-starter-rocketmq/.travis.yml

```diff
@@ -0,0 +1,14 @@
+notifications:
+  email:
+    recipients:
+      - dev@rocketmq.apache.org
+      - angus.aqlu@gmail.com
```

Comment:
Hi, you can remove your personal email, and subscribe rocketmq dev mailing lists, please refer to [contact](http://rocketmq.apache.org/about/contact/)

Reply:
ok, removed it.